pallets/gear/src/weights.rs

```diff
@@ -389,6 +390,16 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
             // Standard Error: 455_985
             .saturating_add(Weight::from_parts(163_069_244, 0).saturating_mul(r.into()))
     }
+    /// The range of component `p` is `[1, 512]`.
+    fn alloc_per_page(p: u32, ) -> Weight {
+        // Proof Size summary in bytes:
+        //  Measured:  `0`
+        //  Estimated: `0`
+        // Minimum execution time: 83_351_000 picoseconds.
+        Weight::from_parts(111_337_720, 0)
+            // Standard Error: 455_985
+            .saturating_add(Weight::from_parts(163_069_244, 0).saturating_mul(p.into()))
```

Comment:
This is very strange number is absolutely equal to the alloc and it's too big.

Reply:
The same. Didn't re-calculated

Reply:
We can calculate required ones and push from local machine, as new weights are coming with release tag

Reply:
Use CI action Benchamrks